frontend/WikiContrib-Frontend/src/query.js

```diff
@@ -839,13 +848,19 @@ export class Query extends Component {
                             }
                           />
                           <Popup
-                            content="Add User"
+                            content=<a href="https://github.com/wikimedia/WikiContrib"
+                              rel="noopener noreferrer" target="_blank">
+                              Contact The maintainers of this Project to learn more about this feature
+                            </a>
                             position="top center"
+                            pinned
                             trigger={
                               <Button
```

Comment:
So, this button I think we can just comment in the code and not show at all to the user.

Reply:
Ok @srish . But I just have one thing to say, if we disable it, it means we won't be able to get feedback from users enquiring about the feature since they won't even know that such a feature exists. I will remove it so that it can be merged, but then when you return we can discuss about it more